src/templates/app/user/authenticate.jsx

```diff
@@ -277,11 +293,17 @@ const Authenticate = () => (
                                 <img className='gr-padding-20' src={it.url_for('images/pages/authenticate/submit_successfully.svg')} />
                                 <h1 className='gr-padding-10'>{it.L('We\'ve received your document number')}</h1>
                                 <p>{it.L('We\'ll process your details within a few minutes and notify its status via email. Next, we\'ll need your proof of address')}</p>
+                                <p>{it.L('We\'ll process your details within a few minutes and notify its status via email. Next, we\'ll need your proof of income')}</p>
```

Comment:
same here, proof of income it not connected to other function and can be triggered separately. Please correct me if I'm not right